lib/form/js/repeatable.js

```diff
@@ -4,257 +4,338 @@
 (function formRepeatable() {
   'use strict';
 
+  // Disabled one-var and func-names for better readability
```

Comment:
`func-names`, for instance, aids in debugging when functions exist within closures (like they do here) and `one-var` ensures we know what's going to be variable at the top of our function scope so there are no surprises